query/src/sessions/session_mgr.rs

```diff
@@ -341,6 +341,11 @@ impl SessionManager {
                     builder.endpoint(&s3_conf.endpoint_url);
                 }
 
+                // Region
+                {
+                    builder.region(&s3_conf.region);
```

Comment:
It would be better to check the region is empty or not:
```
!if se_config.region.is_empty(){
  builder.region(&s3_conf.region);
}
```

Reply:
> It would be better to check the region is empty or not:
> 
> ```
> if !s3_config.region.is_empty(){
>   builder.region(&s3_conf.region);
> }
> ```

opendal will handle all those work.